metron-platform/metron-parsers/src/main/java/org/apache/metron/parsers/bolt/ParserBolt.java

```diff
@@ -118,6 +116,27 @@ protected void initializeStellar() {
     StellarFunctions.initialize(stellarContext);
   }
 
+  private Map<String, Object> getMetadata(Tuple t, boolean readMetadata) {
+    Map<String, Object> ret = new HashMap<>();
+    if(!readMetadata) {
+      return ret;
+    }
```

Comment:
Why is this tied to Kafka?  The ParserBolt is not, or was not before.  The 'kafka-ness' of this should be abstracted away.